projects/common/src/time/time-range.service.ts

```diff
@@ -70,7 +70,18 @@ export class TimeRangeService {
         map(activeRoute => activeRoute.snapshot.queryParamMap),
         distinctUntilChanged((prevParamMap, currParamMap) => this.shouldNotUpdateTimeRange(prevParamMap, currParamMap)),
         map(paramMap => paramMap.get(TimeRangeService.TIME_RANGE_QUERY_PARAM)), // Extract the time range value from it
-        filter((timeRangeString): timeRangeString is string => !isEmpty(timeRangeString)), // Only valid time ranges
+        filter((timeRangeString): timeRangeString is string => {
+          if (isEmpty(timeRangeString)) {
+            /**
+             * This will add the a time param to the url and re-trigger navigation$
```

Comment:
If this wasn't needed before, why is it now? This seems like it would risk refreshing the page

Reply:
I think it was broken before. When you hit the URL with no TR, the default TR is set, but the URL is not updated. This breaks the nav history. 

The logic around page time range fixed this issue because we set a TR explicitly.

Reply:
I tested the changes, and they seem to work well. Not leading to duplicate emits.